app/src/main/java/com/example/ide/data/repository/AIRepository.kt

```
@@ -43,33 +43,39 @@ class AIRepository {
+                AIModelType.QWEN,
+                AIModelType.LONGCAT_AI,
+                AIModelType.DEEPSEEK,
+                AIModelType.Z_MODEL,
+                AIModelType.KIMI -> {
+                    sendChineseFreeModelMessage(model, apiKey, messages)
+                }
+                else -> {
+                    Result.failure(Exception("Model not yet implemented"))
+                }
+            }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)